components/script/dom/window.rs

```diff
@@ -1101,6 +1101,12 @@ impl Window {
         self.layout_chan.send(Msg::AdvanceClockMs(delta, tick)).unwrap();
     }
 
+    pub fn page_zoom(&self, magnification: f32) {
```

Comment:
Also, let's write a comment here about how's the expected usage, so people don't have to grep on the tests.